manifests/vhost.pp

```diff
@@ -2368,7 +2368,7 @@
   concat::fragment { "${name}-serversignature":
     target  => "${priority_real}${filename}.conf",
     order   => 90,
-    content => template('apache/vhost/_serversignature.erb'),
+    content => '  ServerSignature Off',
```

Comment:
POSIX define a [line](https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/basedefs/V1_chap03.html#tag_03_206) as:

> A sequence of zero or more non-\<newline\> characters plus a terminating \<newline\> character.

We must pass complete lines to `content`, otherwise `concat` will generate invalid config (which is probably causing the acceptance tests to fail):

```suggestion
    content => "  ServerSignature Off\n",
```